src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__photo {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin: 0;
+    margin-bottom: 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &__code {
+    font-weight: 400;
+    line-height: 14px;
+    margin: 4px 0 16px;
+    color: #616070;
```

Comment:
Create variables for colors and transition duration as well so it will be the same for all elements